compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/HttpDiagnosticCodes.java

```diff
@@ -62,6 +63,15 @@
             ERROR),
     HTTP_120("HTTP_120", "invalid media-type subtype '%s'", ERROR),
     HTTP_121("HTTP_121", "invalid multiple 'http:RequestContext' parameter: '%s'", ERROR),
+    HTTP_122("HTTP_122", "invalid multiple 'error' parameter: '%s'", ERROR),
+    HTTP_123("HTTP_123", "invalid multiple interceptor type reference: '%s'", ERROR),
+    HTTP_124("HTTP_124", "invalid multiple interceptor resource functions", ERROR),
+    HTTP_125("HTTP_125", "invalid annotation '%s': annotations are not supported for interceptor " +
+            "resource functions", ERROR),
+    HTTP_126("HTTP_126", "invalid resource method return type: expected '" +
+              ALLOWED_INTERCEPTOR_RETURN_UNION + "', but found '%s'", ERROR),
+    HTTP_127("HTTP_127", "invalid resource path: expected default resource path, but found '%s'", ERROR),
```

Comment:
```suggestion
    HTTP_127("HTTP_127", "invalid interceptor resource path: expected default resource path: '[string... s]', but found '%s'", ERROR),
```

Reply:
Fixed in [`45d852c`](https://github.com/ballerina-platform/module-ballerina-http/pull/732/commits/45d852c3608119603f1bf9cd8616c27b76e0c2db)